test/functional-tests/tests/pageActions.js

```diff
@@ -60,6 +62,7 @@ step('Assert url host is <hostName>', async function(hostName) {
 });
 
 step('Assert page navigated back', async function() {
+  waitFor(intervalSecs(5));
```

Comment:
I feel we should spend time looking at what is actually causing the `timeout` rather than adding waits since adding waits will make tests flaky anyways. We want to avoid using waits in our test suites to capture any scenario where we have to wait for right responses.

I agree there will be scenarios where application might demand waits, but IMO for this scenario we can spend some time understanding whats happening behind page navigations to handle it as in Taiko. Thoughts ?